tests/python/test_offline_cache.py

```diff
@@ -558,29 +543,29 @@ def added_files(arch):
 
     def helper1():
         @ti.experimental.real_func
-        def sum(l: ti.i32, r: ti.i32) -> ti.i32:
+        def sum(l: ti.i32, r: ti.i32, R: ti.i32) -> ti.i32:
```

Comment:
Why do we change this test?

Reply:
The re-implemented Offline Cache cache the jit evaluators. The old `sum` generates more than one cache files because of constant folding. To avoid constant folding, let `sum` use the `R` parameter instead of `2`.

Reply:
But after supporting host constant folding, this change will be not needed.

Reply:
I disabled caching the evaluators.